packages/sirius-web/frontend/sirius-web-application/src/graphql/useCreateApolloClient.ts

```diff
@@ -102,12 +104,21 @@ export const useCreateApolloClient = (httpOrigin: string, wsOrigin: string): Apo
     },
   };
 
+  const { data: documentTransforms } = useData(apolloClientOptionsDocumentTransformConfigurersExtensionPoint);
+
+  const documentTransformToLoad = documentTransforms.reduce(
+    (documentTransforms: DocumentTransform, documentTransform: DocumentTransform) =>
+      documentTransforms.concat(documentTransform)
+  );
+
   let apolloClientOptions: ApolloClientOptions<NormalizedCacheObject> = {
     link,
     cache,
     connectToDevTools: true,
     defaultOptions,
+    documentTransform: documentTransformToLoad,
```

Comment:
Why do you need another extension point when configurers just a couple line below can transform as they like the options?

By the way, please write commit messages that explain what they do and not how they do it. Nobody will understand what this extension point means. Write instead that you are simplifying the contribution to the GraphQL subscription of the diagram for custom nodes.

Reply:
I added the extension point because I wanted to make it more explicit but I'll remove it then.
I'll modify the commite message.